docs/sdks/ios/methods.md

```diff
@@ -288,3 +338,37 @@ import CleverPush
 let chatView = CPChatView(frame: CGRect(x: 0, y: 0, width: self.view.frame.size.width, height: self.view.frame.size.height))
 self.view.addSubview(chatView)
 ```
+
+## Story
```

Comment:
Please rename to "Stories" and add 1-2 sentences of description what stories are and add 1-2 screenshots.